src/components/SelectPlan/tests/index.test.tsx

```diff
@@ -33,6 +33,7 @@ describe('components/page/Header', () => {
         <SelectComponent {...props} />
       </Router>
     );
+    // tslint:disable-next-line
     expect(wrapper.find('IndicatorsContainer').prop('options')).toMatchSnapshot(
```

Comment:
What's the linting that's being disabled here? Can we specify a single (or multiple) rule(s)?

Reply:
Added the specific rule